### DISCUSSION TITLE
New Discord Channel!

### DISCUSSION THREAD

**Discussion by mnicely** [NVIDIA/cutlass#1086] [General]
As a means to further improve the user experience and education around CUTLASS, we have created a new Discord channel! Click the [link](https://discord.gg/hrw4sxPE) to join! See you there and we thank you for all of your support :)

**Comment by serach24** [comment]
Hello, I believe the link has expired.

**Reply by hwu36** [reply]
https://discord.gg/nvidiadeveloper

try this one?  it points to the top level nvidia discord, cutlass channel is under it.

**Reply by serach24** [reply]
Yeah, this one works. Thanks!
